### PR TITLE
Add remote Docker policy

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Add support policy around CircleCI remote Docker images.

# Reasons
DOCSS-1268

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/remote-docker-images-support-policy.adoc [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+The CircleCI remote Docker images are based on our Linux VM images with Docker installed for the purposes of providing a remote environment that can execute Docker commands within jobs on the Docker executor.
+
+We aim to support the latest two versions of the Docker Engine that are classified as within Security Support status.
+
+Remote Docker images will be updated when a patch version is released from upstream. Tags will be redirected to the updated images automatically as described in the xref:#tagging[tagging] section of this document. We will announce these releases on our link:https://discuss.circleci.com/[Discuss Forum].
+
+[#tagging]
+== Tagging
+
+We support various tags for the remote Docker environment which allow customers to choose the type of remote Docker environment they require.
```
Comment (rosieyohannan) [inline jekyll/_cci2/remote-docker-images-support-policy.adoc:34]:
```suggestion
We support various tags for the remote Docker environment to allow you to choose the type of remote Docker environment you require.
```

File: jekyll/_cci2/remote-docker-images-support-policy.adoc [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+For the latest major version of Docker:
+
+- `edge`: This image is for testing purposes. It is updated frequently without warning and could include breaking changes. 
+
+- `current`: This image is the current stable image. This is the default if no tag is selected.
+
+- `previous`: Once an `edge` image is promoted to current, the previous `current` image is moved to the `previous` tag.
+
+For the previous major version of Docker, we support a single tag following the format of `dockerXX` - e.g., `docker23` for Docker 23. This tag will point to the latest patch version of the major release, and will be updated if any patch versions are issued by the upstream.
```
Comment (rosieyohannan) [inline jekyll/_cci2/remote-docker-images-support-policy.adoc:44]:
```suggestion
For the previous major version of Docker, we support a single tag following the format of `dockerXX`, for example, `docker23` for Docker 23. This tag will point to the latest patch version of the major release, and will be updated if any patch versions are issued by the upstream.
```

Comment (rosieyohannan) [inline jekyll/_cci2/remote-docker-images-support-policy.adoc:44]:
```suggestion
For the previous major version of Docker, we support a single tag following the format of `dockerXX` - e.g., `docker23` for Docker 23. This tag will point to the latest patch version of the major release, and will be updated if any patch versions are issued upstream.
```

File: jekyll/_cci2/remote-docker-images-support-policy.adoc [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+
+[#critical-cve-patches]
+== Critical CVE patches
+
+When critical CVEs are disclosed that affect the versions of the operating system or software stack in our remote Docker images, we will investigate the impact that this has on our images being used within the CircleCI execution environment. 
+
+In most cases, due to the ephemeral and isolated nature of the containers in the environment, it is not necessary to patch these images. We will always communicate our stance on these disclosures via our Discuss Forum.
```
Comment (rosieyohannan) [inline jekyll/_cci2/remote-docker-images-support-policy.adoc:52]:
```suggestion
In most cases, due to the ephemeral and isolated nature of the containers in the environment, it is not necessary to patch these images. We will always communicate our stance on these disclosures on our link:https://discuss.circleci.com/[Discuss Forum].
```

**Review by BytesGuy — APPROVED** [review]

**Review by rosieyohannan — APPROVED** [review]
🎉

**Pull request merged**